src/content/learn/choosing-the-state-structure.md

```diff
@@ -833,11 +829,12 @@ import { initialTravelPlan } from './places.js';
 function PlaceTree({ id, placesById }) {
   const place = placesById[id];
   const childIds = place.childIds;
+   const isChildIdPresent = childIds && childIds.length > 0 
   return (
     <li>
       {place.title}
-      {childIds.length > 0 && (
-        <ol>
+      {
+      isChildIdPresent && <ol>
```

Comment:
This is an unrelated stylistic change? Let's not do that.

Reply:
👍